runtime/databricks/automl_runtime/forecast/pmdarima/model.py

```diff
@@ -283,7 +310,11 @@ def make_future_dataframe(
         )
         return future_df
 
-    def predict_timeseries(self, horizon: int = None, include_history: bool = True) -> pd.DataFrame:
+    def predict_timeseries(
```

Comment:
Should we create a new function instead of changing this one?

Reply:
I made the df optional